### PR TITLE
fix: polish description on facility profile

### LINKED ISSUE #76: Replace/fix facility profile expander

### PR DESCRIPTION
**Pull request by dylantompkins**
# Checklist

- [x] Have I clearly explained the issue or feature request that this PR addresses?
- [x] Have I provided where new implementations for the PR have been made?
- [x] Did I highlight any areas that may need special attention during the review?
- [x] If applicable, did I specify any setup requirements or dependencies for testing?
- [x] Have I disclosed any known issues or limitations related to this PR? Did I explain why these limitations exist and how they might be addressed in the future?
- [x] Have I provided enough context for someone unfamiliar with the issue or the codebase to understand the changes?
- [x] Have I assigned appropriate reviewers who are knowledgeable about the changes? Did I consider including team members who might be affected or interested in the changes?

# Description

## Summary
Closes #76

The previous implementation of the "view more text" component on the facility profile had a laggy animation which made it feel like something was wrong when you expanded it.  I used an external library's component instead, doing away with the animation.

This is important so that our users are confident in our app. Eliminating that laggy animation takes away the fear that there are performance issues.

## Expectations
- There should only be changes to the description section on the facility profile, everything else should remain the same.
- When clicking the "view more"/"view less" button, the text should just jump between the two sizes, rather than animating.
- There are some minor styling changes, since I'm using a completely different implementation of the component. These styling changes are intended to be trivial. If you think that they are more than trivial, please comment as such.
- Description text is still placeholder. Getting the description from the controller is outside the scope of this PR.

## Additional Comments/Notes
- To view the facility profile description
    1. Load the app at the home/map screen
    2. Zoom out until you see one of the location markers on the map
    3. Click the location marker to open that location's facility profile
    4. Look just under the "Add Review" button on that screen.
    5. The paragraph should be 3 lines, end in a `...`, and have a "view more" button upon loading the facility profile page.
    6. Upon clicking the "view more" button, the paragraph should extend to full length (number of lines depends on length of content, no `...`) and the "view more" button should change to a "view less" button. 
- Enjoying the PR template.Thanks @meganfung38!!

### CODE REVIEW

File: navigation/screens/facility-profile.tsx [hunk 1]
```diff
@@ -19,125 +16,53 @@ import { StarRatingDisplay } from "react-native-star-rating-widget";
...
-      <LinearGradient
-        // Button Linear Gradient
-        colors={["#6da798", "#40a4a9"]}
-        end={{ x: 0.1, y: 0.2 }}
-        style={styles.button}
+    <View
+      style={{
+        overflow: "hidden",
+        backgroundColor: "none",
+        marginHorizontal: "3%",
```
Comment (lindlands) [inline navigation/screens/facility-profile.tsx:44]: Margin for this looks a little small. Can this be increased to have more breathing room?

**Review by lindlands — CHANGES_REQUESTED** [review]

File: navigation/screens/facility-profile.tsx (hunk 1 above)
Comment (dylantompkins) [reply navigation/screens/facility-profile.tsx]: increased to 6

File: navigation/screens/facility-profile.tsx [hunk 2]
```diff
@@ -19,125 +16,53 @@ import { StarRatingDisplay } from "react-native-star-rating-widget";
...
+          Reprehenderit nostrud nostrud ipsum Lorem est aliquip amet voluptate
+          voluptate dolor minim nulla est proident. Nostrud officia pariatur ut
+          officia. Sit irure elit esse ea nulla sunt ex occaecat reprehenderit
+          commodo officia dolor Lorem duis laboris cupidatat officia voluptate.
+          Culpa proident adipisicing id nulla nisi laboris ex in Lorem sunt duis
+          officia eiusmod. Aliqua reprehenderit commodo ex non excepteur duis
+          sunt velit enim. Voluptate laboris sint cupidatat ullamco ut ea
+          consectetur et est culpa et culpa duis.
+        </Text>
+      </ViewMoreText>
```
[the remark this answers is not in the document]
Comment (dylantompkins) [reply navigation/screens/facility-profile.tsx:65]: played with numbers and got it to look better

**Review by JonasSchiessl — CHANGES_REQUESTED** [review]
I feel like the text is very hard to read for the view more and view less. It blends into the background a lot so there isn't much contrast. i would also maybe center it and add a down arrow icon for more and up for less. besides that i like how smooth it is and the performance is a lot better.

**Comment by dylantompkins** [comment]
> I feel like the text is very hard to read for the view more and view less. It blends into the background a lot so there isn't much contrast. i would also maybe center it and add a down arrow icon for more and up for less. besides that i like how smooth it is and the performance is a lot better.

I changed the "view more"/"view less" from text to a button (which I sized down). I think this will solve the readability problems you were stating. I did not add any icons because there's really no presedent for that in the app, and I don't want to make this button unique so that it stands out. The other buttons on this page are arguably more important that this one.

**Review by lindlands — CHANGES_REQUESTED** [review]
- I like the View More as a button. The text, however, seems slightly off center (slightly right). Can you check if a JustifyContent/etc property is working correctly?
- I would also say that "Add Review" button should be changed to a different color (the dark blue accent button) as everything blends together a bit much now that "View More" is the same color.
- Nit: There is more space/margin on the bottom of the description/view more block than the top. Is it easy to center it a bit more? If anything you could add a touch more space from the bottom of "Add Review".

**Review by JonasSchiessl — CHANGES_REQUESTED** [review]
Text does not match the text of the other buttons, its also smaller and the button size isn't consistent either. 
I agree with Lindsay that the text is also off center.

**Comment by dylantompkins** [comment]
image

**Review by JonasSchiessl — APPROVED** [review]
Looks good to me

**Review by lindlands — APPROVED** [review]
Looks great!

**Pull request merged**